### PR TITLE
fix: re-run pr validation

### PR DESCRIPTION
**Pull request by RitamAgrawal**
### What does this PR do?
Runs the validate PR workflow when just the PR description is edited.
### What issues does this PR fix or reference?
https://github.com/forcedotcom/easywriter/issues/173, @W-13522251@

### Functionality Before
Validate PR workflow runs when commits are pushed to the branch.

### Functionality After
In addition to the existing functionality, Now Validate PR workflow also runs when just the PR description is edited with no commits pushed.

### CODE REVIEW

File: .github/workflows/validatePR.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
 name: Validate PR
 on:
   pull_request:
+    types: [opened, reopened, edited]
```
Comment (gbockus-sf) [inline .github/workflows/validatePR.yml:4]: Just an FYI when I did this in the EGPT repo it put open PRs into a weird state.  I got past it by just edit each description and it fired off the GHA, but initially they were stuck waiting on validation.  I think it was just a one time issue.

**Review by gbockus-sf — APPROVED** [review]
LGTM!

**Review by gbockus-sf — COMMENTED** [review]
The type check failures are super weird...shouldn't have anything to do with this change

File: .github/workflows/validatePR.yml (hunk 1 above)
Comment (RitamAgrawal) [reply .github/workflows/validatePR.yml:4]: Okay! I will keep that in mind and Watch out for PRs opened after this one is merged.

**Comment by RitamAgrawal** [comment]
> The type check failures are super weird...shouldn't have anything to do with this change

Re-run is also not helpful here.

**Comment by gbockus-sf** [comment]
> > The type check failures are super weird...shouldn't have anything to do with this change
> 
> Re-run is also not helpful here.

Is it happening on develop locally?

**Comment by RitamAgrawal** [comment]
> The type check failures are super weird...shouldn't have anything to do with this change

I think it's because locally it was using node 14.18.3 instead of v16.15.1

**Pull request merged**